local-dev-first-start-only.sh

```diff
@@ -407,7 +407,7 @@ docker $DOCKER_PARAM exec -u hydro-service hydroshare python manage.py migrate s
 echo
 echo "  - docker exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput"
 echo
-docker $DOCKER_PARAM exec -u hydro-service hydroshare python manage.py migrate --fake-initial --noinput
```

Comment:
local-dev.yml has some bad permissions setup and the hydro-service user can't run manage.py.  I've got some ideas of how to fix it but don't have the time at the moment and it's out of the scope of this issue.  The commands for solar weren't actually doing anything before this change because the solar indexes were built in hs-solr.